src/components/usage-page/content.tsx

```diff
@@ -156,6 +156,50 @@ export default function (version: string, name = "heart", suffix = "") {
 </ion-button>`
       )}
 
+      <h2>Browser support</h2>
+      <p>Ionicons are supported on the browsers listed below:</p>
```

Comment:
We might want to change this since it's a bit repetitive with the caption. I have two suggestions to what it can be:

1. Use Frameworks': "Because Ionic is based on web technologies, it works just as well on desktop browsers as it does on mobile devices."
2. Use MDNs': "Report problems with this compatibility data on [GitHub](https://github.com/ionic-team/ionicons/issues)"

Reply:
Sounds good, thanks!